docs/install-upgrade.md

```diff
@@ -109,15 +177,13 @@ for MongoDB Docker images:
 
 2. Save the file.
 
-3. [Install the operator](#procedure).
-
-### Procedure
+3. [Install the operator](#procedure-using-kubectl).
 
-The MongoDB Community Kubernetes Operator is a [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and a controller.
+#### Procedure using kubectl
```

Comment:
few questions here (I know this context existed before your edits):

- is this procedure for installing the operator from source?
- which repo does step one refer to? This one?
- what is operator-sdk? perhaps a link here would provide appropriate context?

Reply:
Unfortunately I'm not sure of the answers to these as I just moved existing content here. Leaving this open for the tech reviewer if they have context.

Reply:
> is this procedure for installing the operator from source?

correct.
> which repo does step one refer to? This one?

I belive this refers to the community operator repo.

> what is operator-sdk? perhaps a link here would provide appropriate context?

We can skip the `operator-sdk` reference here, it's more of an implementation detail.

Reply:
Thanks both, I incorporated these updates.